source/fuzz/pass_management/repeated_pass_instances.h

```diff
@@ -167,6 +168,7 @@ class RepeatedPassInstances {
   REPEATED_PASS_INSTANCE(ReplaceParamsWithStruct);
   REPEATED_PASS_INSTANCE(SplitBlocks);
   REPEATED_PASS_INSTANCE(SwapBranchConditionalOperands);
+  REPEATED_PASS_INSTANCE(SwapFunctions);
```

Comment:
Because you've made this a *final* pass (i.e., you did:

```
MaybeAddFinalPass<FuzzerPassSwapFunctions>(&final_passes_);
```

in fuzzer.cpp) it should not be added to `repeated_pass_instances`.

I.e., there should be no changes to this file.

Happy to provide you with some more context offline about the difference between repeated and final passes.

Reply:
Reverted changes in `repeated_pass_instances.h`